src/components/interactive/article.tsx

```diff
@@ -0,0 +1,35 @@
+// ----- Imports ----- //
+
+import React, { ReactNode } from 'react';
+import { css } from '@emotion/core';
+
+// ----- Styles ----- //
+
+const Styles = css`
+
+    h1, h2, h3, h4, h5, h6 {
+        margin: 0;
+    }
+
+    .meta__extras {
+        display: none;
+    }
```

Comment:
In the current templates we have very few styles: https://github.com/guardian/mobile-apps-article-templates/blob/master/ArticleTemplates/assets/scss/interactive.scss

It looks like dotcom have some more: https://github.com/guardian/frontend/blob/3095a28e951b63ae993aa3d3fcc59a3cf83afc4b/static/src/stylesheets/head.interactive.scss

This sometimes makes it harder for interactive developers inheriting different styles on different platforms. They usually have to apply app specific styles when there's an `ios` or `android` class on the body. We probably won't have this in apps-rendering so I moved in some of the common app styles they use (we have social links in the topbar so they aren't needed inside `meta__extras`)

It would be really useful if we could share the same styles on dotcom/apps for immersive interactives. @gtrufitt @oliverlloyd

Reply:
Yeah, this would be really good - removing that uncertainty of rendering for Visuals would be 💯

Reply:
If we could have an identical foundation (e.g. a shared stylesheet) of styles that they're working against on both platforms, that would be amazing! Could mean developing against dotcom and having the apps version "just work" most of the time 🤔